flink-end-to-end-tests/test-scripts/test_tpch.sh

```diff
@@ -72,6 +74,7 @@ execution:
   planner: blink
   type: batch
   result-mode: table
+  parallelism: 2
```

Comment:
Do we need this change? It's unrelated to the subject of the commit.

Reply:
It's a supplement to this issue: https://issues.apache.org/jira/browse/FLINK-13441. I think this is also a good place to cover such scenarios, and don't need to open a jira for it.

Reply:
How is this related to the linked issue? Honestly I still don't get what is the purpose of this change.

I disagree this is a good place to add such cases. It's impossible to link this change to the reason why it was introduced. I'm not saying we need a new JIRA issue, but it should at least  be in a separate hotfix commit with at least brief explanation what it does.

Reply:
I explained this change in the commit message if you missed that: 

> Also changed test parallelism to 2 to cover the situation that parallelism is higher than the slot number, since the testing cluster only have one task manager and contains only one slot

I agree this can be isolated to another commit.

Reply:
You are right I missed it. Now I get it. I think this would be a perfect description for the extracted commit.

Reply:
Yeah, I already did the exactly thing in my last commit.